conf/test.conf

```diff
@@ -2,3 +2,11 @@
 # ~~~~~
 play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoModule"
 mongodb.uri = "mongodb://localhost:27018/knolxtest"
+
+# Refresh token for youtube authorization
+# ~~~~~
+youtube.refreshToken = ${?REFRESH_TOKEN}
+
+# Limit of parallel youtube uploads at a time
+# ~~~~~
+youtube.actors.limit = 5
```

Comment:
Update this as per application.conf updates and add new line at the end of the file.